modifiers/allocation/modifier.py

```diff
@@ -461,3 +491,43 @@ def determine_scheduler_instructions(self, v):
             v.timeout = 120
 
         handler[v.scheduler](v)
+
+    def set_mpibind(self, scheduler, base_string):
+        handler = {
+            "slurm": "--mpibind=",
+            "flux": "-o mpibind=",
+            "mpi": "--mpibind=",
+            "lsf": "--mpibind=",
+            "pjm": "--mpibind=",
+        }
+        mpi_string = handler.get(scheduler)
+        if scheduler == "flux":
+            flags = {
+                "v": "verbose:1",
+                "vv": "verbose:2",
+                "on": "on",
+                "off": "off",
+                "greedy:0": "greedy:0",
+            }
+            mpi_end = flags.get(self._usage_mode)
+            return f"{base_string} {mpi_string}{mpi_end}"
+        else:
+            mpi_end = self._usage_mode
+            return f"{base_string} {mpi_string}{mpi_end}"
+
+    executable_modifier("mpibind")
+
+    def mpibind(self, executable_name, executable, app_inst=None):
+        pre_exec = []
+        post_exec = []
+        output_file = "{experiment_run_dir}/{experiment_name}.out"
```

Comment:
(question) is `.out` the output captured by Ramble from running the experiment? And then `parse_mpibind_output.py` is searching for output from `mpibind` that is generated as part of running that experiment? I don't know how mpibind works: are these values determined by `mpibind` itself (i.e. not known before running the experiment)?